src/natcap/invest/carbon.py

```diff
@@ -32,109 +33,144 @@
         },
         "lulc_fut_path": {
             **spec_utils.LULC,
             "projected": True,
+            "projection_units": u.meter,
             "required": "calc_sequestration",
             "about": (
-                "A GDAL-supported raster representing the land-cover of the "
-                "future scenario. If REDD scenario analysis is enabled, this "
-                "should be the reference, or baseline, future scenario "
-                "against which to compare the REDD policy scenario."),
-            "name": "Future Landcover"
+                "A map of land cover for the future scenario. "
+                f"{RASTER_VALUES % 'Carbon Pools'} If run valuation model is "
+                "selected, this should be the reference, or baseline, future "
+                "scenario against which to compare the REDD policy scenario. "
+                f"{REQUIRED_IF_SELECTED % 'calculate sequestration'}"),
+            "name": "future land use/land cover"
         },
         "do_redd": {
             "type": "boolean",
             "required": False,
             "about": (
-                "Check to enable REDD scenario analysis.  This requires three "
-                "Land Use/Land Cover maps: one for the current scenario, one "
+                "Enable REDD scenario analysis. This requires three "
+                "land use/land cover maps: one for the current scenario, one "
                 "for the future baseline scenario, and one for the future "
                 "REDD policy scenario."),
-            "name": "REDD Scenario Analysis"
+            "name": "REDD scenario analysis"
         },
         "lulc_redd_path": {
             **spec_utils.LULC,
             "projected": True,
+            "projection_units": u.meter,
             "required": "do_redd",
             "about": (
-                "A GDAL-supported raster representing the land-cover of the "
-                "REDD policy future scenario.  This scenario will be compared "
-                "to the baseline future scenario."),
-            "name": "REDD Policy"
+                "A map of land cover for the REDD policy scenario. "
+                f"{RASTER_VALUES % 'Carbon Pools'} "
+                f"{REQUIRED_IF_SELECTED % 'REDD scenario analysis'}"),
+            "name": "REDD land use/land cover"
         },
         "carbon_pools_path": {
             "type": "csv",
             "columns": {
-                "lucode": {"type": "integer"},
-                "c_above": {"type": "number", "units": u.metric_ton/u.hectare},
-                "c_below": {"type": "number", "units": u.metric_ton/u.hectare},
-                "c_soil": {"type": "number", "units": u.metric_ton/u.hectare},
-                "c_dead": {"type": "number", "units": u.metric_ton/u.hectare}
+                "lucode": {
+                    "type": "integer",
+                    "about": (
+                        "Land use/land cover code. Every value in the LULC "

```

Comment:
Following the 'spell out the first time' rule in this context may not be possible because the order of displaying these strings in the UG is not guaranteed to be the same. I think for common acronyms (LULC, DEM, AOI, TFA) it should be ok to define them only once for the whole user's guide (like on the data sources page). But also, sometimes it's helpful to write out.

Reply:
At least for me, TFA is not like those others in terms of the recognizability of the acronym.